src/pages/ReimbursementAccount/BankAccountPlaidStep.js

```diff
@@ -103,6 +116,27 @@ class BankAccountPlaidStep extends React.Component {
                         bankAccountID={bankAccountID}
                         selectedPlaidAccountID={selectedPlaidAccountID}
                     />
+                    {Boolean(selectedPlaidAccountID) && !_.isEmpty(lodashGet(this.props.plaidData, 'bankAccounts')) &&
+                        <CheckboxWithLabel
+                            style={styles.mt4}
+                            inputID="acceptTerms"
+                            LabelComponent={() => (
+                                <Text>
+                                    {this.props.translate('common.iAcceptThe')}
+                                    <TextLink
+                                        href="https://use.expensify.com/terms"
```

Comment:
```suggestion
                                        href={CONST.TERMS_URL}
```

And the same for BankAccountManualStep and DebitCardPage